api/handlers/service_offering.go

```diff
@@ -50,6 +52,30 @@ func NewServiceOffering(
 	}
 }
 
+func (h *ServiceOffering) get(r *http.Request) (*routing.Response, error) {
+	authInfo, _ := authorization.InfoFromContext(r.Context())
+	logger := logr.FromContextOrDiscard(r.Context()).WithName("handlers.service-offering.create")
+
+	payload := new(payloads.ServiceOfferingGet)
+	if err := h.requestValidator.DecodeAndValidateURLValues(r, payload); err != nil {
+		return nil, apierrors.LogAndReturn(logger, err, "Unable to decode request query parameters")
+	}
+
+	serviceOfferingGUID := routing.URLParam(r, "guid")
+
+	serviceOffering, err := h.serviceOfferingRepo.GetServiceOffering(r.Context(), authInfo, serviceOfferingGUID)
+	if err != nil {
+		return nil, apierrors.LogAndReturn(logger, err, "failed to get service offering: %s", serviceOfferingGUID)
+	}
+
+	brokerIncludes, err := h.getBrokerIncludes(r.Context(), authInfo, []repositories.ServiceOfferingRecord{serviceOffering}, payload.IncludeBrokerFields, h.serverURL)
```

Comment:
The generic include resolver can be used here, you can see how it works [here](https://github.com/cloudfoundry/korifi/blob/df9925de6b12eeb51cc05d3dd70019250b8e0175/api/handlers/service_plan.go#L70) and [here](https://github.com/cloudfoundry/korifi/blob/df9925de6b12eeb51cc05d3dd70019250b8e0175/api/handlers/service_instance.go#L162)

Reply:
In this case should I replace the implementation on the list handler as well ?